site/content/docs/5.2/forms/layout.md

```diff
@@ -66,6 +66,13 @@ By adding [gutter modifier classes]({{< docsref "/layout/gutters" >}}), you can
 
 More complex layouts can also be created with the grid system.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+The browser generated appearance of the `<select>` dropdown menu does not respect the Orange Design System and cannot be changed due to browser limitations.
```

Comment:
For what I understand here, it's that we can't use any select ? It's a bit confusing to say that there is a replacement that can't be used 😅 + Feel like this will lose people on the concept that the form is fine.

Reply:
Sure. This topic will be discussed with the designers. This callout message will have to be modified consequently.

Reply:
Decided : delete the callout